Makefile

```diff
@@ -103,12 +94,18 @@ ifdef RELEASE
 DEFINES += -DNO_ASSERT -DRELEASE
 endif
 
-LATEST_RELEASE=$(shell git tag | grep RELEASE_ | sort | tail -1)
-# use egrep to count lines instead of wc to avoid whitespace error on Mac
-COMMITS_SINCE_RELEASE=$(shell git log --oneline $(LATEST_RELEASE)..HEAD | egrep -c .)
-ifneq ($(COMMITS_SINCE_RELEASE),0)
-DEFINES += -DBUILDNUMBER=\"$(COMMITS_SINCE_RELEASE)\"
+LATEST_RELEASE = $(shell egrep "define JS_VERSION .*\"$$" src/jsutils.h | egrep -o '[0-9]v[0-9]+')
+ifdef RELEASE_LABEL
```

Comment:
This looks like it'll remove the commits part by default? Was there a reason for this?

It got put in because I got badgered by the community about it - it got really confusing having lots of images all of which said `1v81`.

I don't mind having the binary with the same name, but when it boots, it should really display the version and subversion...